docs/en_US/TrainingService/DLTSMode.md

```diff
@@ -0,0 +1,52 @@
+**Run an Experiment on Deep Learning Training Service**
```

Comment:
shall we use the official name (DLWorkspace) of the project here? or the well known aka. DLTS.

Reply:
Ping @hongzhili for suggestion.

Reply:
Change to DLTS